bullet_train-themes-light/lib/tasks/application.rb

```diff
@@ -148,26 +150,26 @@ def self.release_theme(original_theme_name, args)
       end
 
       def self.install_theme(theme_name)
-        # Grabs the current theme from
-        # def current_theme
-        #   :theme_name
-        # end
-        current_theme_regexp = /(^    :)(.*)/
         current_theme = nil
+        new_content = nil
+        File.open("./app/helpers/application_helper.rb", "r") do |f|
+          msmn = Masamune::AbstractSyntaxTree.new(f.readlines.join)
+          current_theme = msmn.symbols.first[:token]
+          new_content = msmn.replace(type: :symbol, old_token: current_theme, new_token: theme_name)
+        end
+
+        File.open("./app/helpers/application_helper.rb", "w") do |f|
+          f.puts new_content
+        end
```

Comment:
What do you think about using a `Pathname` here? (it would need a `require "pathname"` too), but like this:

```ruby
helper = Pathname.new("./app/helpers/application_helper.rb")

msmn = Masamune::AbstractSyntaxTree.new helper.readlines.join
current_theme = msmn.symbols.first[:token]

helper.write msmn.replace(type: :symbol, old_token: current_theme, new_token: theme_name)
```